### PR TITLE
[14.0][MIG] purchase_order_secondary_unit

### PR DESCRIPTION
**Pull request by SurachetS**
standard migration

### CODE REVIEW

File: purchase_order_secondary_unit/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+To use this module you need to:
+
+#. Go to a *Product > General Information tab*.
+#. Create any record in "Secondary unit of measure".
+#. Set the conversion factor.
+#. Go to *Purchase > Quotation > Create*.
+#. Change quantities in line and secondary unit (produc_qty will be change).
```
Comment (kittiu) [inline purchase_order_secondary_unit/readme/USAGE.rst]:
```suggestion
#. Change secondary qty and secondary uom in line, and quantity (product_qty) will be changed (according to the conversion factor).
```

**Review by kittiu — APPROVED** [review]
Functional test OK.
(Please change the readme as suggested too)

File: purchase_order_secondary_unit/readme/USAGE.rst (hunk 1 above)
Comment (SurachetS) [reply purchase_order_secondary_unit/readme/USAGE.rst]: Updated.

**Comment by tonygalmiche** [comment]
On Runbot, I cannot select a unit in the "Default unit purchase" field : 

image

**Comment by SurachetS** [comment]
> On Runbot, I cannot select a unit in the "Default unit purchase" field :
> 
> image

you need to add a secondary unit at the page bottom
image

**Comment by pedrobaeza** [comment]
Please don't add merge commits in a PR, or this will conflict in the future. Rebase instead.

**Comment by SurachetS** [comment]
> Please don't add merge commits in a PR, or this will conflict in the future. Rebase instead.

ok

**Review by PrapassornS — APPROVED** [review]
Functional Tested LGTM.

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
pre-commit issues, please take a look

**Comment by kittiu** [comment]
@SurachetS can you talk a look at this issue.

Also, there was a bug here, https://github.com/OCA/purchase-workflow/pull/1074 please included when v13 fix is merged.

**Comment by kittiu** [comment]
@SurachetS given all the comments. I think it can be a good idea to do a clean migration again in a new PR. WDYT?

**Review by HviorForgeFlow — APPROVED** [review]
LGTM :+1:

**Pull request merged**